mmv1/products/workbench/Instance.yaml

```diff
@@ -384,6 +384,27 @@ properties:
               values:
                 - VIRTIO_NET
                 - GVNIC
+            - !ruby/object:Api::Type::Array
+              name: accessConfigs
+              immutable: true
+              description: |
+                Optional. An array of configurations for this interface. Currently, only one access
+                config, ONE_TO_ONE_NAT, is supported. If no accessConfigs specified, the
+                instance will have an external internet access through an ephemeral
+                external IP address.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: externalIp
+                    default_from_api: true
```

Comment:
```suggestion
```
`default_from_api: true` will translate to `optional` + `computed` in the provider, and given it's required, we can't and don't need to set this flag